stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/WebSocketServerConnectorListener.java

```diff
@@ -71,8 +71,7 @@ public WebSocketServerConnectorListener(WebSocketServicesRegistry servicesRegist
 
     @Override
     public void onMessage(WebSocketInitMessage webSocketInitMessage) {
-        HTTPCarbonMessage msg = new HTTPCarbonMessage(
-                ((DefaultWebSocketInitMessage) webSocketInitMessage).getHttpRequest());
+        HTTPCarbonMessage msg = ((DefaultWebSocketInitMessage) webSocketInitMessage).getHttpCarbonRequest();
```

Comment:
Shouldn't we consider having this as property at this moment since this is a good time to do it? IMO we should not deal with internal implementation of WebSocketInitMessage in order to take the request.

Reply:
Ack